internal/snmp/testdata/mibs/testmib

```diff
@@ -0,0 +1,22 @@
+TGTEST-MIB DEFINITIONS ::= BEGIN
```

Comment:
Thanks for a clear and working MIB!

Do you agree to change the name (and filename) to something like TELEGRAF-TEST-MIB. IMHO that looks a bit more comprehensive. (Could then also be used for inputs/snmp tests?)

Reply:
The name doesn't matter to me.

I hope we can use it in the other tests too. We could refer to it in the other tests or make copies with changes needed for other tests.

I do want to keep the test files simple so people can quickly understand them. If we extend a single mib to cover all tests, it will need to be complex. I would rather different tests use different mibs if that means we can keep the mibs simple.